src/lib.rs

```diff
@@ -248,6 +253,14 @@ impl ImGui {
     pub fn get_time(&self) -> f32 { unsafe { sys::igGetTime() } }
     pub fn get_frame_count(&self) -> i32 { unsafe { sys::igGetFrameCount() } }
     pub fn get_frame_rate(&self) -> f32 { self.io().framerate }
+    /// Get current window's size in pixels
+    pub fn get_window_size(&self) -> (f32, f32) {
```

Comment:
This function should be part of `Ui` instead `ImGui`, because it should only be called while an imgui frame is being prepared. If it's called at the wrong time, the whole application will immediately abort (not even a panic).

Reply:
Ok! Now I understand better why methods in Ui and UmGui are separated. Will fix it shortly (and add a few other wrapper functions in the mean time).